frontend/public/components/edit-yaml.jsx

```diff
@@ -641,6 +652,11 @@ export const EditYAML_ = connect(stateToProps)(
                 {t('public~View sidebar')}
               </Button>
             ) : null;
+          const hoverLink = (
+            <Button type="button" variant="link" isInline onClick={this.toggleHover}>
```

Comment:
This is probably better as a checkbox. We should get UX input.

Reply:
@yuvalgalanti @matthewcarleton would you mind adding some guidance here? Thanks @spadgett

Reply:
@spadgett @yuvalgalanti Replaced the link with a checkbox

Reply:
![Screenshot 2023-05-03 at 11 57 45 AM](https://user-images.githubusercontent.com/35978579/235971483-6f55a145-b204-428c-8d42-76eccf2dae51.png)